### DISCUSSION TITLE
Update chevron in mistica-web

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#919] [Proposals]
actualmente en mistica-web se está usando un chevron custom en las listas.

Dos alternativas:
1. O usar el chevron de mistica-icons tal y como está.
2. Hacer un pequeño ajuste de grosor en los chevrones regular de mistica-icons

actual
[mistica-web.vercel.app/?path=/story/components-lists--row-list-story](https://mistica-web.vercel.app/?path=/story/components-lists--row-list-story)

mistica-icons
[mistica-5dmc3ksbm-tuentisre.vercel.app/?path=/story/components-lists--row-list-story](https://mistica-5dmc3ksbm-tuentisre.vercel.app/?path=/story/components-lists--row-list-story)

**Comment by HectorSancho** [comment]
Pruebas del chevron: https://www.figma.com/design/MLbPCnxqEYh7aXmSMcvSnm/Chevron?node-id=16-638&t=GQkikpSZd8OWaaY2-1
